src/Business Foundation/App/NoSeries/src/Setup/NoSeriesSetupImpl.Codeunit.al

```diff
@@ -137,6 +136,22 @@ codeunit 305 "No. Series - Setup Impl."
         exit(NoSeriesSingle.MayProduceGaps());
     end;
 
+    procedure SetNoSeriesLineFilters(var NoSeriesLine: Record "No. Series Line"; NoSeriesCode: Code[20]; StartingDate: Date)
+    var
+        NoSeries: Codeunit "No. Series";
+        NoSeriesLine2: Record "No. Series Line";
```

Comment:
See errors below, basically the sorting of variables must be fixed, Record comes before codeunit.